common/setups/rasr/util/decode.py

```diff
@@ -0,0 +1,43 @@
+__all__ = [
+    "BaseRecognitionParameters",
+    "PriorArgs",
+    "SearchJobArgs",
+    "Lattice2CtmArgs",
+    "ScorerArgs",
+]
+
+from dataclasses import dataclass
+from typing import Any, Dict, List, Optional, TypedDict
+
+from sisyphus import tk
+
+from ..config.am_config import TransitionTdp, SilenceTdp
+
+
+class BaseRecognitionParameters(TypedDict):
+    am_scales: List[float]
+    lm_scales: List[float]
+    pronunciation_scales: List[float]
+    transition_tdp: List[TransitionTdp]
+    silence_tdp: List[SilenceTdp]
+
```

Comment:
am scale is not important, we always use equal to one. What matters more is tdp scale. I would substitute `am_scales` with `tdp_scales`

Reply:
`prior_scales` are also missing